drivers/ethernet/eth_nxp_enet.c

```diff
@@ -672,6 +711,10 @@ static int eth_nxp_enet_init(const struct device *dev)
 		enet_config.miiMode = kENET_MiiMode;
 	} else if (config->phy_mode == NXP_ENET_RMII_MODE) {
 		enet_config.miiMode = kENET_RmiiMode;
+#ifdef ENET_HAS_AVB
+	} else if (config->phy_mode == NXP_ENET_RGMII_MODE) {
+		enet_config.miiMode = kENET_RgmiiMode;
+#endif /* ENET_HAS_AVB */
```

Comment:
You could do this in order to avoid using `#ifdef`
```
} else if (COND_CODE_1(ENET_HAS_AVB,
	   (config->phy_mode == NXP_ENET_RGMII_MODE), (0))) {
	enet_config.miiMode = kENET_RgmiiMode;
}
```

Reply:
Used when possible.